src/lamp_py/performance_manager/flat_file.py

```diff
@@ -36,14 +38,23 @@ class S3Archive:
         "lamp", "subway-on-time-performance-v1"
     )
     INDEX_FILENAME = "index.csv"
+    VERSION_KEY = "rpm_version"
+    RPM_VERSION = "1.0.0"
 
 
-def dates_to_update(db_manager: DatabaseManager) -> Set[datetime]:
+def dates_to_update(
```

Comment:
this func now returns two sets; one is the set of dates that need to be archived, one that has old metadata and needs to be deleted.

the latter set should always be empty, since any data under an old version _should_ also be in our rds but it felt safer to do it this way.